manage.py

```
@@ -2,9 +2,11 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
+import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)